<launch>

  <arg name="mission_file" default="~/scrimmage/scrimmage/missions/straight.xml"/>  
  <arg name="entity_name" default="uav_entity"/>
  <arg name="entity_id" default="1"/>
  <arg name="max_contacts" default="100"/>
  
  <node pkg="scrimmage_ros"
        name="entity_1"
        type="entity"
        output="screen">
    <param name="mission_file" value="$(arg mission_file)" type="str"/>
    <param name="entity_name" value="$(arg entity_name)" type="str"/>
    <param name="entity_id" value="$(arg entity_id)" type="int"/>
    <param name="max_contacts" value="$(arg max_contacts)" type="int"/>
  </node>
  
</launch> 
